airtime_mvc/application/models/Show.php

```diff
@@ -974,10 +974,7 @@ public static function &getFullCalendarEvents($p_start, $p_end, $p_editable=fals
         foreach ($shows as &$show) {
             $options = array();
 
-            //only bother calculating percent for week or day view.
-            if (intval($days) <= 7) {
-                $options["percent"] = Application_Model_Show::getPercentScheduled($show["starts"], $show["ends"], $show["time_filled"]);
-            }
+            $options["percent"] = Application_Model_Show::getPercentScheduled($show["starts"], $show["ends"], $show["time_filled"]);
```

Comment:
Needed to make this change because the calendar now does not re-request the data if you go from month->week view. Therefore, we must pass this "percent scheduled" data along with any show events. Performance hit is pretty small.